site.yml.sample

```diff
@@ -1,52 +1,88 @@
 ---
 # Defines deployment design and assigns role to server groups
 
+- hosts: all 
+  gather_facts: false
+  tasks:
+    # If we can't get python2 installed before any module is used we will fail
+    # so just try what we can to get it installed
+    - name: Check for python2
+      stat:
+        path: /usr/bin/python
+      ignore_errors: yes 
+      register: systempython2
+    # Try to install python2 on Ubuntu Xenial and possibly others
```

Comment:
I think we can remove this comment since the name of the task is pretty explicit